nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/resources/conf/parameter-context-flow-description.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The flow.xml format is being removed for NIFI-11772 under PR #7661, so the test should be changed to use the JSON serialized version.

Reply:
replaced with json version